supervisor/docker/interface.py

```diff
@@ -162,6 +162,11 @@ def _install(
                     "Available space in /data is: %s GiB",
                     free_space,
                 )
+            elif err.status_code == 429:
+                _LOGGER.info(
+                    "You are now ratelimited by Docker Hub, for more details see "
```

Comment:
Your IP address has made too many requests to Docker Hub which activated a rate limit. For more details see X